src/devices/tuya.ts

```diff
@@ -1616,6 +1617,7 @@ const definitions: DefinitionWithExtend[] = [
                 '_TZ3210_mja6r5ix',
             ]),
             tuya.whitelabel('MiBoxer', 'FUTC11ZR', 'Outdoor light', ['_TZB210_zmppwawa']),
+            tuya.whitelabel('TechToy', 'TS0505B', 'TechToy Smart Bulb RGB 9W E27 ZigBee', ['_TZ3210_iw0zkcu8']),
```

Comment:
```suggestion
            tuya.whitelabel('TechToy', '_TZ3210_iw0zkcu8', 'Smart bulb RGB 9W E27', ['_TZ3210_iw0zkcu8']),
```

`TS0505B` is too generic, so let's use this instead